backend/core/validations/user_validation/sign-in-validation.js

```diff
@@ -0,0 +1,8 @@
+const mailValidation = require('./email-validation.js');
+const passValidation = require('./password-validation.js');
+const loginValidation = require('./login-validation.js');
+
+function validateSignIn(loginRequest) {
+  return !!((mailValidation.validateEmail(loginRequest.login) || loginValidation.validateLogin(loginRequest.login)) && passValidation.validatePassword(loginRequest.password));
```

Comment:
What's the purpose of !! ?

Reply:
check input data before database request

Reply:
https://ru.stackoverflow.com/questions/188946/%D0%94%D0%B2%D0%BE%D0%B9%D0%BD%D0%BE%D0%B5-%D0%BE%D1%82%D1%80%D0%B8%D1%86%D0%B0%D0%BD%D0%B8%D0%B5